composer/loggers/tensorboard_logger.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Log to `Tensorboard <https://www.tensorflow.org/tensorboard/>`_."""
+
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+from torch.utils.tensorboard import SummaryWriter
```

Comment:
This will need to be a conditional import to get the dev tests to pass

Reply:
good catch. done